fab/environments.yml

```diff
@@ -165,40 +165,45 @@ icds:
         num_processes: 1
       CaseSearchToElasticsearchPillow:
         num_processes: 1
-      CaseToElasticsearchPillow:
-        num_processes: 1
       DefaultChangeFeedPillow:
         num_processes: 1
       DomainDbKafkaPillow:
         num_processes: 1
-      FormSubmissionMetadataTrackerPillow:
-        num_processes: 1
       GroupPillow:
         num_processes: 1
       GroupToUserPillow:
         num_processes: 1
-      kafka-ucr-main:
-        num_processes: 1
       kafka-ucr-static-awc-location:
-        num_processes: 1
-      kafka-ucr-static-cases:
         num_processes: 4
+      kafka-ucr-static-cases:
+        num_processes: 8
       kafka-ucr-static-ccs_record_cases:
-        num_processes: 4
+        num_processes: 8
       kafka-ucr-static-ccs_record_cases_monthly:
-        num_processes: 4
+        num_processes: 8
       kafka-ucr-static-child_cases_monthly:
-        num_processes: 1
+        num_processes: 8
       kafka-ucr-static-child_health_cases:
-        num_processes: 4
+        num_processes: 8
       kafka-ucr-static-person_cases:
+        num_processes: 8
+    '10.247.24.30': # pillow1
+      CaseToElasticsearchPillow:
         num_processes: 4
+      FormSubmissionMetadataTrackerPillow:
+        num_processes: 2
+      kafka-ucr-static-forms-A:
+        num_processes: 8
+      kafka-ucr-static-forms-B:
+        num_processes: 8
+      kafka-ucr-main:
```

Comment:
I'm not sure that we need this pillow in its current form.

We should probably have a new pillow that ignores any icds-cas submissions since all UCRs should be static for icds-cas. will confirm with @sheelio when he comes back from vacation